internal/tools/aws/session.go

```diff
@@ -14,9 +17,26 @@ func NewAWSSessionWithLogger(config *aws.Config, logger log.FieldLogger) (*sessi
 		return nil, err
 	}
 
-	awsSession.Handlers.Send.PushFront(func(r *request.Request) {
-		logger.Debugf("%s: %s%s\n%s", r.HTTPRequest.Method, r.HTTPRequest.URL.Host, r.HTTPRequest.URL.RawPath, r.Params)
+	awsSession.Handlers.Complete.PushFront(func(r *request.Request) {
+		loggingMessage := fmt.Sprintf("%s.%s %s %s: %s%s  %q", r.ClientInfo.ServiceID, r.Operation.Name, r.HTTPRequest.Method,
+			r.HTTPResponse.Status, r.HTTPRequest.URL.Host, r.HTTPRequest.URL.RawPath, marshallParams(r.Params))
```

Comment:
Because `marshallParams` is only used once and chooses a non-standard error handling path, I think I'd prefer to see the `json` marshaling done without the helper function just so that it's clearer that if the JSON marshaling fails for whatever reason that the error text will be included as part of `loggingMessage`.

I don't feel strongly about this, though, so if you disagree feel free to keep it. However, if you choose to keep the helper function, please rename it to `marshalParams` to match `json.Marshal`.